packages/toolpad-components/src/DataGrid.tsx

```diff
@@ -242,7 +292,7 @@ const DataGridComponent = React.forwardRef(function DataGridComponent(
   );
   React.useEffect(() => handleColumnOrderChange.clear(), [handleColumnOrderChange]);
 
-  const rowsInput = rowsProp || EMPTY_ROWS;
+  const rowsInput = rowsProp || DEFAULT_ROWS;
```

Comment:
is there a way to set the default value here automatically according to the `defaultValue` in `argTypes`?

Reply:
i mean you're already passing `DEFAULT_ROWS` to `defaultValue` below, so not 100% how this logic works but maybe it could be set automatically as the initial value of that prop

Reply:
i just think it's a bit unpredictable if the default value for the props is not guaranteed to match the value we set in the `argTypes`... but if it's not something we're doing yet we can do it later separately.

Reply:
ah, i just checked and i see that the default value already works like i was describing, i wasn't sure. so this is kind of a fallback? 
would keeping `EMPTY_ROWS` as the fallback value work?
if not, and this fallback is necessary, we can keep it.

Reply:
>  would keeping `EMPTY_ROWS` as the fallback value work? 

Yep, I think `EMPTY_ROWS` should remain as well as the fallback for cases when the `defaultValue` isn't the value of the `prop` yet